src/Benchotron/UI/Console.purs

```diff
@@ -94,9 +93,9 @@ runBenchmarkConsole benchmark = do
   lift $ do
     stderrWrite "\n"
     noteTime \t -> "Finished at: " <> t <> "\n"
-  return r
+  pure r
   where
-  noteTime f = now >>= toLocaleTimeString >>= (stderrWrite <<< f)
+  noteTime f = now >>= show >>= (stderrWrite <<< f)
```

Comment:
Well, I managed to implement toLocaleTimeString as 
```purescript
import Date.JSDate as JSD
import Date.DateTime.Instant as DDI
nowToString = JS.toTimeString <$> JS.fromDateTime <$> DDI.toDateTime <$> now
```
Doesn't look right to me at least from the design perspective.